### PR TITLE
Bug 1969374: document how to update domain for image registry in versions <4.8

### PR DESCRIPTION
**Pull request by Fedosin**
/label platform/openstack

### CODE REVIEW

File: docs/user/openstack/known-issues.md [hunk 1]
```diff
@@ -92,6 +92,29 @@ To recover from a scenario where one or multiple master nodes are in failed stat
...
+I0303 17:32:15.603731      14 controller.go:291] object changed: *v1.Config, Name=cluster (status=true): 
+E0303 17:32:15.610609      14 controller.go:330] unable to sync: unable to sync storage configuration: Failed to authenticate provider client: Authentication failed, requeuing
+I0303 17:32:15.644501      14 controller.go:291] object changed: *v1.Config, Name=cluster (status=true): 
+E0303 17:32:15.650130      14 controller.go:330] unable to sync: unable to sync storage configuration: Failed to authenticate provider client: Authentication failed, requeuing
+```
+
+This issue was fixed in 4.8, which means that all new deployments work correctly and update image registry configuration in the case of credentials change. For older versions, and for clusters that have been upgraded to 4.8, you need to manually clean the config and delete the incorrect values. 
+
+To do so, execute `oc edit configs.imageregistry.operator.openshift.io/cluster` and manually remove `domain` or `domainID` in the `.spec.storage.swift` section. Alternatively you can use the `oc patch` command:
```
Comment (mandre) [inline docs/user/openstack/known-issues.md:111]: Should it be 'remove' or 'update'?

Comment (Fedosin) [reply docs/user/openstack/known-issues.md:111]: It should be removed, because the operator will insert the new value automatically.

**Review by mandre — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**